lib/galaxy/tool_util/verify/__init__.py

```diff
@@ -31,7 +31,7 @@
 try:
     from PIL import Image
 except ImportError:
-    pass
+    Image = None  # type: ignore[assignment]
```

Comment:
I changed this to `pass` on purpose since in the rest of this file we are not handling the case where `Image` is None (as we instead do in `lib/galaxy/util/image_util.py`), so a `NameError: name 'Image' is not defined` exception is better than a `AttributeError: 'NoneType' object has no attribute 'open'` one.

No need to change it back anyway, and thanks for the other fix (backport of https://github.com/galaxyproject/galaxy/pull/17749 ).